WordOfTheDay.PerformanceCheck/WordOfTheDay.PerformanceCheck/Request.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Diagnostics;
+
+namespace WordOfTheDay.PerformanceCheck
+{
+    public class Request
+    {
+        int AmountOfQueries { get; set; }
+        string WordsFile { get; set; }
+        public Request(int amountOfQueries, string wordsFile)
+        {
+            AmountOfQueries = amountOfQueries;
+            WordsFile = wordsFile;
+        }
+        public List<string> GetWords()
+        {
+            var words = File.ReadAllText(WordsFile).Split().ToList();
+
+            return words;
+        }
+        public static string GetRandomWord(List<string> words)
+        {
+            Random random = new();
+
+            string randomString = words[random.Next(0, words.Count - 1)];
+
+            return randomString;
+        }
+        public static string GetRandomEmail()
+        {
+            Random random = new();
+            string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
```

Comment:
Inconsistency in the style of typization. If you use "var", so use "var" everywhere it is possible, or do not use "var" at all.